### PR TITLE
[Backend] Contact Us Delete Cron Job added

### PR DESCRIPTION
**Pull request by Aniket-508**
## Issue that this pull request solves

 Closes: #697 

### Brief description of what is fixed or changed

- Added Contact Us CronJob email template that will be sent to Admin when deletion takes place.
- The CronJob for Contact Us would execute after every 60 days.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

NA

## Other information

This PR is raised under JWoC.

### CODE REVIEW

**Comment by Aniket-508** [comment]
@Kajol-Kumari Please have a look at this.

File: backend/utility/emailTemplates.js [hunk 1]
```diff
@@ -55,3 +56,13 @@ module.exports.ContactUsMailTemplate = (adminName, req) => {
   `;
   return emailContent;
 };
+
+module.exports.ContactUsCronJobMailTemplate = (adminName) => {
+  const emailContent = `
+  <h4>Hi ${adminName},<br>
+  Contact Us Data has been removed !
```
Comment (Kajol-Kumari) [inline backend/utility/emailTemplates.js:63]: Please change it to `The Contact Us Form data of last 2 month has been removed!`

File: backend/index.js [hunk 2]
```diff
@@ -42,6 +43,41 @@ cronJob('0 0 2 * *', async () => {
   }
 });
 
+// Running Contact Us cronjob after every 1 month - 0 0 1 * *
+cronJob('0 0 1 * *', async () => {
```
Comment (Kajol-Kumari) [inline backend/index.js:47]: Can we please make it for `2 months` in order to keep the things consistent

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aniket-508 Please do the requested minor changes required.
Rest, looks good to go

**Comment by Aniket-508** [comment]
@Kajol-Kumari Can you check it now?

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Pull request merged**